release.js

```diff
@@ -55,12 +55,34 @@ module.exports = async function ({ github, context, inputs, callApi }) {
 
   // TODO: What if PR was closed, reopened and then merged. The draft release would have been deleted!
 
-  await callApi({
-    endpoint: 'release',
-    method: 'PATCH',
-    body: {
-      version: version,
-      releaseId: id,
-    },
-  })
+  try {
+    await callApi({
+      endpoint: 'release',
+      method: 'PATCH',
+      body: {
+        version: version,
+        releaseId: id,
+      },
+    })
+
+    const syncInput = inputs['sync-major']
+    const syncMajor =
+      syncInput === 'true' || syncInput === 'True' || syncInput === 'TRUE'
+
+    if (syncMajor) {
+      const major = version.split('.')[0]
+
+      await run('git', ['push', 'origin', `:refs/tags/${major}`])
+      await run('git', [
+        'tag',
+        '-fa',
+        `"${major}"`,
+        '-m',
+        `"Update tag ${major}"`,
+      ])
+      await run('git', ['push', 'origin', `--tags`])
+    }
```

Comment:
if you want to avoid manual parsing consider using the semver npm package which is a more reliable alternative to manually parsing and interpreting version numbers unless you're only doing very basic manipulation

Reply:
I second using `semver` too. You could do

```js
const { major, minor } = require('semver').parse(version)
```